src/patternfly/components/LabelGroup/label-group.scss

```diff
@@ -0,0 +1,98 @@
+  --pf-c-label-group__label--MarginBottom: 0;
+  --pf-c-label-group--m-vertical__label--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-label-group__label--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-label-group__label--MaxWidth: 18ch;
+
+  // label group close
+  --pf-c-label-group__close--MarginTop: calc(var(--pf-global--spacer--xs) * -1);
+  --pf-c-label-group__close--MarginBottom: calc(var(--pf-global--spacer--xs) * -1);
+  --pf-c-label-group--m-vertical__close--MarginRight: calc(var(--pf-global--spacer--sm) * -1);
+  --pf-c-label-group--m-vertical__close--MarginLeft: calc(var(--pf-global--spacer--sm) * -1);
+  --pf-c-label-group--m-vertical__close--c-button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label-group--m-vertical__close--c-button--PaddingLeft: var(--pf-global--spacer--sm);
+
+  // label
+  --pf-c-label-group__list-item--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-label-group__list-item--MarginBottom: var(--pf-global--spacer--xs);
+
+  &.pf-m-category {
+    padding-top: var(--pf-c-label-group--m-category--PaddingTop);
+    padding-right: var(--pf-c-label-group--m-category--PaddingRight);
+    padding-bottom: var(--pf-c-label-group--m-category--PaddingBottom);
+    padding-left: var(--pf-c-label-group--m-category--PaddingLeft);
+    background-color: var(--pf-c-label-group--m-category--BackgroundColor);
+    border: var(--pf-c-label-group--m-category--BorderWidth) solid var(--pf-c-label-group--m-category--BorderColor);
+    border-radius: var(--pf-c-label-group--m-category--BorderRadius);
+  }
+
+  &.pf-m-vertical {
+    --pf-c-label-group__label--MarginRight: 0;
+    --pf-c-label-group__label--MarginBottom: var(--pf-c-label-group--m-vertical__label--MarginBottom);
+    --pf-c-label-group__close--MarginTop: 0;
+    --pf-c-label-group__close--MarginLeft: var(--pf-c-label-group--m-vertical__close--MarginLeft);
+    --pf-c-label-group__close--MarginBottom: 0;
+    --pf-c-label-group__close--MarginRight: var(--pf-c-label-group--m-vertical__close--MarginRight);
+
+    &.pf-c-label-group,
+    .pf-c-label-group__list {
+      flex-direction: column;
+      align-items: flex-start;
+    }
+
+    .pf-c-label-group__close .pf-c-button {
+      --pf-c-button--PaddingLeft: var(--pf-c-label-group--m-vertical__close--c-button--PaddingLeft);
+      --pf-c-button--PaddingRight: var(--pf-c-label-group--m-vertical__close--c-button--PaddingRight);
+    }
+  }
+}
+
+.pf-c-label-group__list {
+  margin-right: var(--pf-c-label-group__list--MarginRight);
+  margin-bottom: var(--pf-c-label-group__list--MarginBottom);
+}
+
+.pf-c-label-group,
+.pf-c-label-group__list {
+  display: inline-flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.pf-c-label-group__list-item {
+  display: inline-flex;
+  margin-right: var(--pf-c-label-group__list-item--MarginRight);
+  margin-bottom: var(--pf-c-label-group__list-item--MarginBottom);
+}
+
+.pf-c-label-group__label {
```

Comment:
i think its confusing that this is called `label-group__label` , since its unlike the actual label component, maybe it could be called `text` ?

Reply:
I'm in favor of keeping it consistent unless there is a good reason not to. I think the element "label" applies to the group, so it's fine as the group's label. Alternatively I would also be in favor of this and the chip group renaming `__label` to `__title`, as that seems a little more appropriate. I don't think we should call it text, and that's usually just the text node of some other element (as in we would have `.pf-c-label-group__label > .pf-c-label-group__label-text`). WDYT, still want to change it?

Reply:
Okay yeah if it means updating the chip group too, that would be a breaking change, so keeping it as the group's label is ok with me